src/api-config/api-config.service.ts

```diff
@@ -31,4 +31,18 @@ export class ApiConfigService {
   isProduction(): boolean {
     return this.get<string>('NODE_ENV') === 'production';
   }
+
+  get dbPoolUrl(): string {
+    return `${this.get<string>('DATABASE_CONNECTION_POOL_URL')}?pgbouncer=true`;
+  }
+
+  get readDbPoolUrl(): string {
+    const readDbPoolUrl = this.config.get<string>(
```

Comment:
I believe config.get throws if the key isn't found (although it probably shouldn't), should this be like a try/catch or something?

Reply:
This is using the build-in NestJS Config service which does not appear to throw, only [return `undefined`](https://github.com/nestjs/config/blob/master/lib/config.service.ts#L98-L123).